### PR TITLE
[GPT2] Add generation unwrapping protection

### PR DESCRIPTION
**Pull request by texasmichelle**
Every once in a while, GPT2-Inference will crash with this error:
```
Illegal instruction (core dumped)
```

This change adds unwrapping protection in case something goes wrong during generation.

### CODE REVIEW

File: Models/Text/GPT2/GPT2.swift [hunk 1]
```diff
@@ -154,7 +154,7 @@ public class GPT2 {
             randomCategorialLogits: logits.squeezingShape(at: 1),
             sampleCount: 1)
 
-        let id = Int32(seed[0][0])!
+        guard let id = Int32(seed[0][0]) else { return "" }
```
Comment (BradLarson) [inline Models/Text/GPT2/GPT2.swift:157]: In the cases where a seed isn't generated correctly, is it preferred to quietly return an empty string over throwing a GPT2Error of some kind?

Comment (texasmichelle) [reply Models/Text/GPT2/GPT2.swift]: that's a good point. We can throw here and continue from main instead.

**Comment by texasmichelle** [comment]
I'm still seeing `Illegal instruction (core dumped)`, but this will hopefully aid in eventually locating the root cause.

**Comment by compnerd** [comment]
@texasmichelle I'm not sure I follow how this helps in locating the root cause?  This hides the error, previously, we would actually hit the unreachable codepath, allowing us to understand whats going on in in a debugger
